.gitignore

```diff
@@ -1 +1,2 @@
 coverage
+.DS_Store
```

Comment:
ooh what's this?

Reply:
@rrabinovitch it's the thing that shows up when you mess around with your files via Finder GUI and not the terminal. `.gitignore` is where you want those. Or else they drive you insane.

Reply:
word, thanks for explaining!